app/ide-desktop/client/tests/cloudWorkflow.spec.ts

```diff
@@ -0,0 +1,124 @@
+/** @file A series of tests designed for testing GUI behavior in Cloud. */
+
+import { expect } from 'playwright/test'
+import { closeWelcome, createNewProject, loginAsTestUser, test } from './electronTest'
+
+// A test controlling if project session logs aren't empty. Currently skipped due to unconsistency of session logs
+test.skip('Session logs', async ({ page }) => {
+  await loginAsTestUser(page)
+  await closeWelcome(page)
+
+  // Switching to a private cloud folder and creating new project
+  await page.getByRole('button', { name: 'Cloud', exact: true }).click()
+
+  await createNewProject(page)
+
+  // Executing
+  await page.getByLabel('Write All').click()
+
+  // Returning back to the data catalog
+  const dataCatalogTab = page.getByRole('tab', { name: 'Data Catalog' })
+  await expect(dataCatalogTab).toBeVisible()
+  await dataCatalogTab.click()
+
+  // Finding all of the 'New projects'
+  const projects = await page
+    .getByTestId('drive-view')
+    .getByText(/New Project \d+/)
+    .all()
+  const numbered = await Promise.all(
+    projects.map(async (p) => {
+      const text = await p.innerText()
+      const num = parseInt(text.replace('New Project ', ''), 10)
+      return { locator: p, num }
+    }),
+  )
+  const newest = numbered.reduce((a, b) => (a.num > b.num ? a : b)).locator
+  await newest.click()
+
+  await page.getByLabel('Sessions').click()
+
+  // Navigating into the last log
+  try {
+    const firstRow = page.locator('div.flex.flex-row.gap-4.rounded-2xl.p-2').first()
+    const showLogsButton = firstRow.getByRole('button', { name: /show logs/i })
+    await showLogsButton.click()
+  } catch {
+    console.log('No session logs available')
+  }
+
+  await expect(page.getByText('Starting Language Server')).toBeVisible()
+})
+
+// Test designed to see, if removing a member from Enso organisation shows imediately in GUI
+test('Remove Member', async ({ page }) => {
+  await loginAsTestUser(page)
+  await closeWelcome(page)
+
+  // Navigating into the Members settings tab
+  await page.getByLabel('User Settings').click()
+  await page.getByRole('button', { name: 'Settings' }).click()
+
+  await expect(page.getByText('Settings for')).toBeVisible()
+
+  try {
+    await page.getByRole('button', { name: 'Members', exact: true }).click()
+  } catch {
+    test.skip(true, 'Not a member of an organization.')
+    return
+  }
+
+  const rows = page.locator('table tbody tr')
```

Comment:
prefer `getByRole('row')` if that works. we have rules to disallow `.locator` in dashboard tests because it doesn't match with how a user would perceive these elements